### DISCUSSION TITLE
Enable Workflow Run Failure Alerts (Self host)

### DISCUSSION THREAD

**Discussion by guedesfelipe** [hatchet-dev/hatchet#728] [Q&A] [answered]
How I can configure alerts in self host like cloud portal?
![image](https://github.com/user-attachments/assets/9e6bfb15-51c9-4a09-bdf2-83f39c450cfb)
![image](https://github.com/user-attachments/assets/fd2092bf-b8e9-4709-b226-f3c98ceb315e)

**Comment by abelanger5** [comment, marked as answer]
Hey @guedesfelipe, the docs on this are a bit sparse, but you basically need to create a slack app with incoming webhooks support and the following env: https://docs.hatchet.run/self-hosting/configuration-options#tenant-alerting-configuration

"Email Groups" requires a set of Postmark environment variables (unfortunately no SMTP support yet), happy to provide some pointers there but the slack app will be much easier to set up.

**Reply by guedesfelipe** [reply]
Hmmmm nice @abelanger5 !!

At the moment we can't configure Slack for this unfortunately... is there any other way for me to be notified when a workflow fails? (especially when no worker is active)

Webhook would be interesting for example

I tried to configure the sentry, but without success either

**Reply by abelanger5** [reply]
Unfortunately we only support email and slack for workflow failures, but we'd be open to adding a webhook for failure events! Will track this in a separate issue. 

> I tried to configure the sentry, but without success either

If this was the Go SDK, could you open a new issue if you were seeing an error with the integration?

**Reply by guedesfelipe** [reply]
> Unfortunately we only support email and slack for workflow failures, but we'd be open to adding a webhook for failure events! Will track this in a separate issue.

Perfect! Perfect! I'll wait for this webhook feature! It will be very good! Would you prefer that I open an issue about this?

> f this was the Go SDK, could you open a new issue if you were seeing an error with the integration?

Actually, I was using the Python SDK and trying to check the workflow failures within Sentry, but I don't think it would work, right?

Would the sending to Sentry only be for the worker done through the SDK? (https://docs.hatchet.run/self-hosting/configuration-options#alerting-configuration)
